src/eventshub/src/component/SearchItem/VenueSearchItem.tsx

```diff
@@ -60,10 +60,10 @@ const VenueSearchItem = ({ hit }: VenueSearchItemProps) => {
       </div>
       <FlexBox direction='col' className='er-col-span-7 md:er-col-span-8 lg:er-col-span-9 last:children:er-mb-0'>
         <FlexBox direction='row' justifyContent='between'>
-          <Heading as='h3' className='er-m-0'>
+          <Heading as='h3' className='er-m-0 er-flex-grow'>
             {hit.space_name}
           </Heading>
-          <Paragraph  className='er-m-0'>
+          <Paragraph className='er-m-0 er-pl-5 er-text-right er-min-w-[160px]'>
```

Comment:
Set a min-w so long headings don't wrap the name differently for different results.